src/style.css

```diff
@@ -0,0 +1,33 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--0.stars__star {
+  background-image: url(images/star.svg);
```

Comment:
The 'background-image' property is already set in line 20, you can remove this line